### ISSUE TITLE
[Bug Report] Picker components cannot move next input on Safari

### ISSUE THREAD

**Issue by ochi51** [ElemeFE/element#11913]
### Element UI version
2.4.3

### OS/Browsers version
All mac OS and iOS / Safari version 11.1.1 (12605.2.8)

### Vue version
2.5.16

### Reproduction Link
[https://jsfiddle.net/ochi51/j6n57t0L/22/](https://jsfiddle.net/ochi51/j6n57t0L/22/)

### Steps to reproduce
- Move cursor to the top input element
- Push tab
- Push tab

### What is Expected?
Cursor position is the bottom input element

### What is actually happening?
Cursor position is the top input element

**Comment by wacky6** [comment]
Likely related to tabindex issues: https://github.com/ElemeFE/element/issues/11757
